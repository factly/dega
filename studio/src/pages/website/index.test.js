import React from 'react';
import { shallow } from 'enzyme';
import { act } from '@testing-library/react';
import Website from '.';
import '../../matchMedia.mock';

describe('website component', () => {
  describe('snapshot testing', () => {
    it('should render the component', () => {
      let component;
      act(() => {
        component = shallow(<Website />);
      });
      expect(component).toMatchSnapshot();
      act(() => component.unmount());
    });
  });
});
