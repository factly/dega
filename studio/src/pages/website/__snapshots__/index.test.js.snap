// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`website component snapshot testing should render the component 1`] = `
<div>
  <HelmetWrapper
    defer={true}
    encodeSpecialCharacters={true}
    title="Website"
  />
  <Row
    gutter={
      Array [
        16,
        16,
      ]
    }
  >
    <Col
      span={12}
    >
      <Link
        to="/website/general"
      >
        <ForwardRef
          hoverable={true}
        >
          <Meta
            avatar={
              <ForwardRef(InternalAvatar)
                icon={<ForwardRef(DeploymentUnitOutlined) />}
                shape="circle"
                size="default"
                style={
                  Object {
                    "backgroundColor": "transparent",
                    "color": "#ffb41f",
                  }
                }
              />
            }
            description="Basic Site Details and Site Meta Data"
            title="General"
          />
        </ForwardRef>
      </Link>
    </Col>
    <Col
      span={12}
    >
      <Link
        to="/website/branding"
      >
        <ForwardRef
          hoverable={true}
        >
          <Meta
            avatar={
              <ForwardRef(InternalAvatar)
                icon={
                  <ForwardRef(TagsTwoTone)
                    twoToneColor="#51bbf6"
                  />
                }
                shape="circle"
                size="default"
                style={
                  Object {
                    "backgroundColor": "transparent",
                  }
                }
              />
            }
            description="Update site logos, icons and design tokens"
            title="Branding"
          />
        </ForwardRef>
      </Link>
    </Col>
    <Col
      span={12}
    >
      <Link
        to="/website/menus"
      >
        <ForwardRef
          hoverable={true}
        >
          <Meta
            avatar={
              <ForwardRef(InternalAvatar)
                icon={
                  <ForwardRef(CompassTwoTone)
                    twoToneColor="#7b2feb"
                  />
                }
                shape="circle"
                size="default"
                style={
                  Object {
                    "backgroundColor": "transparent",
                  }
                }
              />
            }
            description="Setup Menus"
            title="Navigation"
          />
        </ForwardRef>
      </Link>
    </Col>
    <Col
      span={12}
    >
      <Link
        to="/website/code-injection"
      >
        <ForwardRef
          hoverable={true}
        >
          <Meta
            avatar={
              <ForwardRef(InternalAvatar)
                icon={
                  <ForwardRef(ApiTwoTone)
                    twoToneColor="#30cf43"
                  />
                }
                shape="circle"
                size="default"
                style={
                  Object {
                    "backgroundColor": "transparent",
                  }
                }
              />
            }
            description="Add code to your site"
            title="Code Injection"
          />
        </ForwardRef>
      </Link>
    </Col>
    <Col
      span={12}
    >
      <Link
        to="/website/analytics"
      >
        <ForwardRef
          hoverable={true}
        >
          <Meta
            avatar={
              <ForwardRef(InternalAvatar)
                icon={
                  <ForwardRef(FundTwoTone)
                    twoToneColor="#fb2d8d"
                  />
                }
                shape="circle"
                size="default"
                style={
                  Object {
                    "backgroundColor": "transparent",
                  }
                }
              />
            }
            description="View Analytics for your site"
            title="Analytics"
          />
        </ForwardRef>
      </Link>
    </Col>
  </Row>
</div>
`;
